src/java/fr/paris/lutece/plugins/forms/web/admin/MultiviewFormResponseDetailsJspBean.java

```diff
@@ -473,8 +476,13 @@ public String doProcessWorkflowAction( HttpServletRequest request ) throws Acces
             {
                 boolean bIsAutomaticAction = Boolean.FALSE;
 
-                workflowService.doProcessAction( nIdFormResponse, FormResponse.RESOURCE_TYPE, nIdAction, formResponse.getFormId( ), request, locale,
+                List<ResourceHistory> actionHistoryResourceList = workflowService.doProcessAction( nIdFormResponse, FormResponse.RESOURCE_TYPE, nIdAction, formResponse.getFormId( ), request, locale,
                         bIsAutomaticAction, getUser( ) );
+                
+                if ( CollectionUtils.isNotEmpty( actionHistoryResourceList ) && actionHistoryResourceList.stream( ).noneMatch( ah -> NumberUtils.INTEGER_MINUS_ONE == ah.getStatus( ) ) )
```

Comment:
we should not filter on the status, in fact it is not really a question of success or failure, it is the choice on return of the action of the next main state, or of the alternative state (not necessarily a failure).